lib/store/inventory/orders_products.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Store.Inventory.Orders_Products do
```

Comment:
I think the module should be called `Order_Product` (and the file `order_product.ex`) and the schema should be called `orders_products`. At least this is the convention that we follow on other projects.